.travis.yml

```diff
@@ -40,12 +40,6 @@ matrix:
 
   # Integration tests
   include:
-    - env: RAILS_VERSION=4.0
```

Comment:
It is good that we are leaving 4.2 not only for ourselves but because this is open source.

Reply:
I'm not sure what you mean here.  Are you saying you are glad I didn't get rid of 4.2, or are you suggesting we should keep 4.0?

Reply:
The former.